osu.Game/Screens/Menu/LogoVisualisation.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.ES30;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Batches;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.OpenGL.Vertices;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Graphics.Shaders;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using System;
+
+namespace osu.Game.Screens.Menu
+{
+    internal class LogoVisualisation : Drawable, IHasAccentColour
+    {
+        private Bindable<WorkingBeatmap> beatmap = new Bindable<WorkingBeatmap>();
+        private Color4 barColour;
+
+        private const int index_change = 5;
+        private const int bar_length = 1200;
+        private const int bars_per_visualizer = 250;

```

Comment:
🤔 I counted 50 bars between big bars (bass) and they were 5 big bars, so I used that